README.md

```diff
@@ -222,7 +238,9 @@ Here are blog posts related to ***snap*** by the team:
 
 
 ### Examples
-There are interesting examples of using snap in every plugin repository. For the full list of plugins, review the [plugin catalog](docs/PLUGIN_CATALOG.md). There are also [Examples](examples/README.md) that demonstrates snap configurations and usages.
+There are some more complex examples of Snap examples and configuration files under the [Examples folder](examples/README.md). We welcome you adding to it.
```

Comment:
I'd leave something about adding to it. Not sure how I feel about how that is worded though.